jenkins-pipelines/rolling-upgrade-with-null-inserts.jenkinsfile

```diff
@@ -0,0 +1,14 @@
+#!groovy
+
+// trick from https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/43
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+rollingUpgradePipeline(
+    backend: 'gce',
+    base_versions: ['4.1'],
```

Comment:
@roydahan 
Now that version `4.2` is official, it's not better to run on top of `4.2`

Reply:
Basicailly you're right now, but for now we can keep it 4.1 till later phases of 4.3.
In any case, I'll send another PR to have the base_versions as a job parameter for all rolling upgrades jobs.